code/datums/dna.dm

```diff
@@ -596,3 +556,88 @@
 	return value
 
 /////////////////////////// DNA HELPER-PROCS
+
+/mob/living/carbon/human/proc/something_horrible(ignore_stability)
+	if(!has_dna()) //shouldn't ever happen anyway so it's just in really weird cases

```

Comment:
usually happens for a tick when a mob is gibbed tbh.

Reply:
well, if the blue moon hits and this happens, there wont be a runtime then